drivers/java/src/test/java/com/rethinkdb/gen/TransformArray.java

```diff
@@ -2,43 +2,24 @@
 // Do not edit this file directly.
 // The template for this file is located at:
 // ../../../../../templates/Test.java
-package gen;
+package com.rethinkdb.gen;
 
 import com.rethinkdb.RethinkDB;
-import com.rethinkdb.gen.exc.*;
-import com.rethinkdb.gen.ast.*;
-import com.rethinkdb.ast.ReqlAst;
-import com.rethinkdb.model.MapObject;
+import com.rethinkdb.TestingFramework;
+import com.rethinkdb.gen.ast.MakeArray;
+import com.rethinkdb.gen.ast.Union;
 import com.rethinkdb.model.OptArgs;
 import com.rethinkdb.net.Connection;
-import com.rethinkdb.net.Cursor;
-import junit.framework.TestCase;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertArrayEquals;
+import org.junit.After;
```

Comment:
If these additions are made only in this file, it's going to get clobbered when the tests are regenerated from the template